tests/test_audio_48000kHz.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from facefusion.ffmpeg import restore_audio
+
+class TestRestoreAudio(unittest.TestCase):
```

Comment:
we are not using classes in this project - just use methods

Reply:
I fixed classes to methods